_schedule/talks/2022-10-17-09-30-t0-opening-remarks.md

```diff
@@ -3,8 +3,9 @@ accepted: true
 category: talks
 date: 2022-10-17 09:30:00-07:00
 end_date: 2022-10-17 09:45:00-07:00
+group: talks
 layout: session-details
-permalink: /talks/opening-remarks-monday/
+permalink: /talks/opening-remarks/
```

Comment:
We need to keep the separate slugs for each day of lightning talks. I'll make a separate PR to adjust that.

Reply:
Fixed!